maven-wrapper/src/test/java/org/apache/maven/wrapper/SystemPropertiesHandlerTest.java

```diff
@@ -31,23 +31,23 @@
 import java.util.Map;
 import java.util.Properties;
 
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 
-public class SystemPropertiesHandlerTest
+class SystemPropertiesHandlerTest
```

Comment:
We create test API docs. Please let all classes and methods be public. Tests fail because of this.

Reply:
from parent 38 it will be ok 😄 
https://issues.apache.org/jira/browse/MPOM-340

Reply:
Thanks, missed that! Will then change to approved and wait with merging until the new parent version is merged.

Reply:
Great; thanks for the thorough review; let me know if/when I need to rebase.
I'm expecting the parent change to also clear up the pipeline issue with mvn site.

Reply:
Actually, not so great. You will need to redo most of it b/c parent v39 requires a new style (which will cause a LOT of git merge conflicts): https://github.com/apache/maven-wrapper/pull/81

Maybe you just want to update here to v38 first before I do the v39 update?

Reply:
Hmm I just updated the parent to v38 and ran spotless:apply, but that seems to have given a lot more changes than I was expecting based on your comment.

Worst case I can always recreate the changes whenever the main branch is stable, as most of the changes are automated anyway. Let me know how best to proceed! :)

Reply:
Huh... yeah, maybe that is better.
Can you recreate this PR and https://github.com/apache/maven-wrapper/pull/80 after I merge the v39 update as you suggested? That would be awesome!

Reply:
Sure; I'll force push to #61 and #80 when #81 and it's [possible refactor followup](https://github.com/apache/maven-wrapper/pull/81#issuecomment-1372125545) are done.